packages/manager/modules/anthos/src/dashboard/components/add-storage/component.js

```diff
@@ -6,6 +6,9 @@ export default {
     goBack: '<',
     serviceName: '<',
     storageUsage: '<',
+    trackClick: '<',
+    trackPage: '<',
+    addStorageHitTracking: '<',
```

Comment:
```suggestion
    addStorageHitTrackingId: '<',
```

Reply:
Thanks for suggestion, i am not sure the `...Id` is relevant.
Id is something that can have more than one value.

So `addStorageHitTracking` is good, what do you think ?